pages/home.html

```diff
@@ -21,20 +27,18 @@
     Bite-sized
   </h3>
   <p>
-    Explorations are easy to create and quick to play. They can be published in
-    the exploration gallery, shared on social media, or embedded on your
-    webpage.
+    Easy to create and quick to play, explorations can also be published in the exploration gallery, shared on social media, or embedded on your webpage.
```

Comment:
Totally agree. I've also brought this question to the oppia-chat and all of the responses incline to the original version. I focused too much on writing good and catchy sentences, I should pay attention to the fact that non-native English speaker will read it too! Next time I'll try to use short, simple expression instead :smile: